src/main/java/ch/yth2021/charjar/discord/module/FileReadHelper.java

```diff
@@ -0,0 +1,26 @@
+package ch.yth2021.charjar.discord.module;
+
+import ch.yth2021.charjar.discord.bot.Application;
+import org.apache.commons.io.IOUtils;
+
+import java.io.InputStream;
+import java.nio.charset.StandardCharsets;
+import java.util.HashMap;
+
+public class FileReadHelper {
+    private static final HashMap<String, String[]> cache = new HashMap<>();
+
+    public static String[] getLinesAsArray(String filename) {
+        return cache.computeIfAbsent(filename, FileReadHelper::getLinesFromFile);
```

Comment:
what is the cache for? is it necessary?

Reply:
The Modules get called on every message event, if i dont cache the file the hole file gets read in on every message... not performing well. I Just save all the words in memory once. In The old version there was also a FileStream open the entire time

Reply:
how does the cache work when we now have 2 different files? swear words and nice words? which file gets put in the cache?